### PR TITLE
Added push to deply workflow

### PR DESCRIPTION
**Pull request by patilvikasj**

### CODE REVIEW

File: .github/workflows/push-to-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    name: New tag
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Build
+      run: |
+        npm install
+        npm run build
+    - name: WordPress Plugin Deploy
+      uses: 10up/action-wordpress-plugin-deploy@master
```
Comment (Nikschavan) [inline .github/workflows/push-to-deploy.yml]: use action `Nikschavan/action-wordpress-plugin-deploy@develop` for now until there is a resolution to https://github.com/10up/action-wordpress-plugin-deploy/issues/1

File: .travis.yml [hunk 2]
```diff
@@ -7,25 +8,24 @@ notifications:
   email:
     on_success: never
     on_failure: change
-  slack: bsf-bots:RDGCoMDDxnlCvnUfvhq8pQ36
+    slack: bsf-bots:RDGCoMDDxnlCvnUfvhq8pQ36
```
Comment (Nikschavan) [inline .travis.yml]: You can remove this notification for slack as we no longer use this.

File: composer.json [hunk 3]
```diff
@@ -6,5 +6,19 @@
...
+    "extra": {
+        "hooks": {
+            "pre-commit": [
+            "echo committing as $(git config user.name)",
+            "sh bin/block-commits-with-merge-conflict.sh"
+            ]
+        }
+    },
+    "scripts": {
+        "post-install-cmd": "vendor/bin/cghooks add --ignore-lock",
```
Comment (Nikschavan) [inline composer.json]: For these scripts to work we will also need package `brainmaestro/composer-git-hooks` check our [Astra's composer.json](https://github.com/brainstormforce/astra/blob/master/composer.json) for it's implementation example.

While you are at it, update version use the same packages from Astra's composer config and add `name` and `description` in the composer.json to make the syntax valid.

File: .github/workflows/push-to-deploy.yml (hunk 1 above)
Comment (patilvikasj) [reply .github/workflows/push-to-deploy.yml]: Done with the changes.

File: .travis.yml (hunk 2 above)
Comment (patilvikasj) [reply .travis.yml]: Done.

File: composer.json (hunk 3 above)
Comment (patilvikasj) [reply composer.json]: Updated composer.json

**Pull request merged**